pkg/ncproxy/nodenetsvc/v1/doc.go

```diff
@@ -0,0 +1,11 @@
+package v1
+
+// This package contains the proto and compiled go files for the node
+// network service.
+//
+// A mock service under `mock` is used for unit testing the various services
+// used for ncproxy.
+//
+// The mock service is compiled using the following command:
+//
+// mockgen -source="nodenetsvc.pb.go" -package="nodenetsvc_mock" > mock\nodenetsvc_mock.pb.go
```

Comment:
If this is a package comment it should be above the package declaration

Reply:
Updated. Sorry I forgot to push a new commit :(